yarn.lock

```diff
@@ -1769,6 +2149,162 @@
   resolved "https://registry.yarnpkg.com/@types/zxcvbn/-/zxcvbn-4.4.0.tgz#fbc1d941cc6d9d37d18405c513ba6b294f89b609"
   integrity sha512-GQLOT+SN20a+AI51y3fAimhyTF4Y0RG+YP3gf91OibIZ7CJmPFgoZi+ZR5a+vRbS01LbQosITWum4ATmJ1Z6Pg==
 
+"@webassemblyjs/ast@1.8.5":
```

Comment:
Would you know why we're installing `webassemblyjs?`

Reply:
```
=> Found "@webassemblyjs/ast@1.8.5"
info Reasons this module exists
   - "webpack" depends on it
   - Hoisted from "webpack#@webassemblyjs#ast"
   - Hoisted from "webpack#@webassemblyjs#wasm-edit#@webassemblyjs#ast"
   - Hoisted from "webpack#@webassemblyjs#wasm-parser#@webassemblyjs#ast"
   - Hoisted from "webpack#@webassemblyjs#helper-module-context#@webassemblyjs#ast"
[...]
```

I guess it's part of some WASM bundling feature of Webpack.

Reply:
👍 ok